modules/persistent-storage-csi-azure-file-cross-sub-dynamic-pre-provisioning-pv-pvc-procedure.adoc

```diff
@@ -0,0 +1,86 @@
+// Module included in the following assemblies:
+//
+// * storage/container_storage_interface/persistent_storage-csi-azure-file.adoc
+//
+:_mod-docs-content-type: PROCEDURE
+[id="persistent-storage-csi-azure-file-cross-sub-dynamic-pre-provisioning-pv-pvc-procedure_{context}"]
+= Pre-provisioning across subscriptions for Azure File by creating a PV and PVC:
+
+.Prerequisites
+* Installed Azure cluster with the service principal or managed identity as Azure identity
+
+* Access to another subscription in the tenant with the cluster  
+
+* Logged in to the Azure CLI
+
+.Procedure
+. For the desired existing Azure File share, record the resource group, storage account, storage account key, and Azure File name. These values are used for the next steps.
+
+. Create a secret for the persistent volume parameter `spec.csi.nodeStageSecretRef.name` by running the following command:
++
+[source, terminal]
+----
+$ oc create secret generic azure-storage-account-{storageaccount-name}-secret --from-literal=azurestorageaccountname="<azure-storage-account-name>" --from-literal azurestorageaccountkey="<azure-storage-account-key>" --type=Opaque <1>
+----
+<1> Where `<azure-storage-account-name>` and `<azure-storage-account-key>` are the Azure storage account name and key respectively that you recorded in Step 1.
```

Comment:
Maybe [this format](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#callouts-in-code-blocks)?

```suggestion
[source, terminal]
----
$ oc create secret generic azure-storage-account-{storageaccount-name}-secret --from-literal=azurestorageaccountname="<azure-storage-account-name>" --from-literal azurestorageaccountkey="<azure-storage-account-key>" --type=Opaque
----
+
--
where:
 `<azure-storage-account-name>` and `<azure-storage-account-key>` are the Azure storage account name and key respectively that you recorded in Step 1.
--
```